index.js

```diff
@@ -0,0 +1,146 @@
+  const gridOrder = document.querySelector('.book-order');
+  gridOrder.innerHTML = '';
+
+  const listOfOrder = document.createElement('ul');
+  Object.keys(orders).forEach( bookName => {
+    var detail = document.createElement('li');
+    detail.textContent = `${bookName} for ${orders[bookName]} books`;
+    listOfOrder.appendChild(detail);
+  })
+
+  const orderLine = document.createElement('p');
+  orderLine.textContent = listOfOrder;
+
+  displayPrice();
+
+  gridOrder.append(listOfOrder);
+
+}
+
+const displayPrice = () => {
+  const gridPrice = document.querySelector('.book-price');
+  var TotalPrice = 0;
+  Object.values(orders).forEach ( amount => {
+    TotalPrice += amount*80;
+  })
+  switch (Object.keys(orders).length) {
+    case 2:
+      TotalPrice *= 0.95;
+      break;
+    case 3:
+      TotalPrice *= 0.9;
+      break;
+    case 4:
+      TotalPrice *= 0.8;
+      break;
+    case 5:
+      TotalPrice *= 0.75;
+      break;
+  }
+  gridPrice.textContent = `${TotalPrice} baht`;
+
+}
+
+
+const displayBook = () => {
+  const gridContainer = document.querySelector('.book-info')
+
+  gridContainer.innerHTML = "";
+
+  books.forEach( book => {
+    const bookItem = document.createElement('div');
+    bookItem.classList.add('book');
+
+    const bookImage = document.createElement('div');
+    bookImage.classList.add('book-image');
+    bookImage.style.backgroundImage = `url(${book.background})`;
+
+    const bookTitle = document.createElement('h1');
+    bookTitle.textContent = book.name;
+
+    bookImage.appendChild(bookTitle);
+
+    const bookDetail = document.createElement('div');
+    bookDetail.classList.add('book-detail');
+
+    const bookName = document.createElement('p');
+    const spanName = document.createElement('span');
+    spanName.textContent = 'Name: ';
+    bookName.appendChild(spanName);
+    bookName.innerHTML += book.name;
+
+    const bookDetail2 = document.createElement('div');
+    bookDetail2.classList.add('book-detail2');
+
+    const bookPrice = document.createElement('p');
+    bookPrice.textContent = `Price: ${book.price}`;
+    bookPrice.style.flexBasis = '50%';
+    const bookAmount = document.createElement('p');
+    bookAmount.textContent = `Available: ${book.amount}`;
+    bookDetail2.append(bookPrice, bookAmount);
+
+    bookDetail.append(bookName, bookDetail2);
+
+
+    const bookOrder = document.createElement('div');
+    bookOrder.classList.add('book-button');
+
+    var quantity = 0;
+
+    const orderButton = document.createElement('button');
+    orderButton.innerHTML = 'Add to cart';
+    orderButton.type = 'submit';
+
+    const select = document.createElement('select');
+
+    for (var i=0; i <= book.amount; i++){
+      var option = document.createElement('option');
+      option.value = i;
+      option.textContent = i;
+      select.appendChild(option);
+    }
+
+    select.onchange = function() {
+      quantity = this.value;
+    }
+
+    orderButton.onclick = function () {
```

Comment:
i think this action could create a new function like onAddToCart outside` displayBook` function and you can call function onAddToCart add send data to funtion